.github/workflows/package-workflow.yml

```diff
@@ -0,0 +1,38 @@
+name: E4E Packaging Workflow
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    env:
+      PROJECT_NAME: explorer-for-endevor
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
```

Comment:
May I propose to use more modern versions of Node? I know, that 16.x versions are also possible here, maybe we can specify only 16,14 and 12? What do you think?

Reply:
Done!